File: Laboratórios/lab3_c++/Pessoa.cpp
#include "Pessoa.hpp"
#include <iostream>
using namespace std;

string Pessoa::getNome()
{
    return nome;
}
void Pessoa::setNome(string nome)
{
    this->nome = nome;
}

string Pessoa::getEndereco()
{
    return endereco;
}
void Pessoa::setEndereco(string endereco)
{
    this->endereco = endereco;
}

File: Laboratórios/Java/lab07_java/src/lab07_java/Pessoa.java
package lab07_java;

public class Pessoa {

    private String nome;
    private String cpf;
    private Pessoa proximo = null;

    public Pessoa(String nome, String cpf) {
        this.nome = nome;
        this.cpf = cpf;
    }

    public void getDetalhes(){
        System.out.println("\tNome: " + nome
		+ "\n\tCPF: " + cpf + "\n");
    }
    
	public String getNome() {
		return this.nome;
	}

	public void setNome(String nome) {
		this.nome = nome;
	}

	public String getCpf() {
		return this.cpf;
	}

	public void setCpf(String cpf) {
		this.cpf = cpf;
	}
	public Pessoa getProximo() {
		return this.proximo;
	}

	public void setProximo(Pessoa proximo) {
		this.proximo = proximo;
	}    
}
	